automated-comments/go/hamming/return_on_error_case.md

```diff
@@ -0,0 +1,8 @@
+Go is a programming language where returning as soon as possible and multiple times from a function or method 
+is practiced a lot. It keeps code readable and maintainable.
+
+Usually one would handle edge cases first and error cases as soon as they occur by returning. This practice is
+nicely explained in the blog post [Align the happy path to the left edge](https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88) by Mat Ryer.
+You can also watch the talk he gave on the toppic.
+  
+Can you apply this to your solution?
```

Comment:
I'd like to not use questions from the auto-mentoring comments, since questions presume answers, and if the person comments with an answer the auto-mentoring will not have any way of responding again.

To tighten this up, maybe:

> In Go, it's conventional to use multiple returns, returning as soon as possible from a function. Typically edge cases and error cases are handled as early as possible, leaving the happy path aligned to the left edge. For a nice explanation of this practice, see [Align the happy path to the left edge](https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88) by Mat Ryer, or watch the talk that he gave on the same topic.